src/external/samba.m4

```diff
@@ -126,7 +126,7 @@ int main(void)
     samba_minor_version=`printf '#include <samba/version.h>\nSAMBA_VERSION_MINOR' | $CPP $SMBCLIENT_CFLAGS -P -`
     samba_release_version=`printf '#include <samba/version.h>\nSAMBA_VERSION_RELEASE' | $CPP $SMBCLIENT_CFLAGS -P -`
     AC_MSG_NOTICE([Samba version: $samba_major_version $samba_minor_version $samba_release_version])
-    if test $samba_major_version -ge 4 -a $samba_minor_version -ge 8 ; then
+    if test $samba_major_version -ge 4 -a $samba_minor_version -ge 7 -a $samba_release_version -ge 4 ; then
```

Comment:
Hi,

thanks for working through the Samba history. I think there is an issue with the check because it would fail e.g. for version 4.8.[0-3], the same for 4.9 etc. Can you fix this?

bye,
Sumit

Reply:
Btw, I think this is the reason for the CI failures as well.

Reply:
Good catch. I think the following changes cover all combinations, including samba >= 5.0.